x-pack/plugins/infra/public/pages/metrics/metric_detail/components/page_error.test.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
Enzyme works just fine, especially if we need to play with component internals and state. In this case we don't need to do that so I believe we could use react-testing-library. What do you think?

Reply:
That's a good point, thank you. I agree, I realize that we don't actually need to change the internal state there - just to test if the hook is called and the text error message is correct. I updated it 👍